packages/jest-circus/src/__mocks__/testUtils.ts

```diff
@@ -13,7 +13,7 @@ import crypto from 'crypto';
 import {sync as spawnSync, ExecaReturns} from 'execa';
 import {skipSuiteOnWindows} from '@jest/test-utils';
 
-const CIRCUS_PATH = require.resolve('../../build');
+const CIRCUS_PATH = require.resolve('../../build/');
```

Comment:
I think we don't need to add `/` after `build`

Reply:
It'll resolve to the index.js path. It could probably be without `build` as well, and it'll use `main`

Reply:
adding the `/` wasn't strictly necessary, it's just a result of churn because I changed the import path to something else and then back again

doesn't make a difference either way, it'll resolve to index